corehq/apps/registration/forms.py

```diff
@@ -565,7 +588,26 @@ def clean_email(self):
 
     def clean(self):
         cleaned_data = super(AdminInvitesUserForm, self).clean()
+
+        if (('tableau_role' in cleaned_data or 'tableau_group_indices' in cleaned_data)
+        and self.can_edit_tableau_config):
```

Comment:
should this be a negative case, as in `and not and self.can_edit_tableau_config`?

Reply:
yes! good catch thank you [c98633e](https://github.com/dimagi/commcare-hq/pull/34678/commits/c98633e223c61b4eaf5f4767ebe091e4c97e2912)